services/s3/src/main/java/software/amazon/awssdk/services/s3/S3CrtAsyncClientBuilder.java

```diff
@@ -352,6 +352,15 @@ default S3CrtAsyncClientBuilder retryConfiguration(Consumer<S3CrtRetryConfigurat
      */
     S3CrtAsyncClientBuilder futureCompletionExecutor(Executor futureCompletionExecutor);
 
+    /**
+     * Configure whether to disable this client's usage of Session Auth for S3Express buckets and reverts to using conventional
+     * SigV4 for those.
+     *
+     * @param disableS3ExpressSessionAuth whether S3ExpressSessionAuth should be disabled
```

Comment:
Instead of "S3ExpressSessionAuth" we should just repeat what you said above: "Session Auth for S3Express..."